api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -27,6 +28,12 @@ DbWorkspace findByWorkspaceNamespaceAndFirecloudNameAndActiveStatus(
   @Query("SELECT distinct w.workspaceNamespace, w from DbWorkspace w")
   Set<String> findAllWorkspaceNamespaces();
 
+  @Query(value = "SELECT workspace_id FROM workspace limit :limit", nativeQuery = true)
+  List<BigInteger> findTopDbWorkspaceIds(@Param("limit") Integer limit);
+
```

Comment:
I am going to modify this query to return all workspaces that have NOT changed (last_modified <= last_export)